slots/utils.py

```diff
@@ -68,12 +68,12 @@ def check_time_slot_for_age_group(member_id, slot_datetime, library):
     member_age = Member.objects.filter(member_id=member_id)[0].age
     if member_age < 16:
```

Comment:
this logic seems a little out of place here. we are adding some business logic based on a feature of a member class. Instead, we can add methods inside member object itself, like `isAdult()` etc and call these methods from the utility classes.